modules/querying/pages/accumulators.adoc

```diff
@@ -1447,7 +1447,7 @@ MapAccum<SetAccum<INT>, INT> # illegal
 GroupByAccum<INT a, STRING b, MaxAccum<INT> maxs, ListAccum<ListAccum<INT>> lists>
 ----
 
-. ArrayAccum: Unlike the other accumulators in this list, where nesting is optional, nesting is mandatory for ArrayAccum. See the xref:querying:accumulators.adoc#_arrayaccum[ArrayAccum] section.
+. ArrayAccum: Unlike the other accumulators in this list, where nesting is optional, nesting is mandatory for ArrayAccum. See the xref:querying:accumulators.adoc#arrayaccum[ArrayAccum] section.
```

Comment:
https://docs.tigergraph.com/gsql-ref/current/querying/accumulators#_arrayaccum is the link for array accum - by default Antora generates the anchors with a preceding underscore.

Reply:
Weird, they showed up as typos/not found for me in Intellij